conan.install.profile

```diff
@@ -0,0 +1,23 @@
+[settings]
+os=Windows
+os_build=Windows
+arch=x86_64
+arch_build=x86_64
+compiler=Visual Studio
+compiler.version=16
+compiler.toolset=v142
+build_type=Release
+
+libpq:compiler.version=17
+libpq:compiler.toolset=v143
+openssl:compiler.version=17
+openssl:compiler.toolset=v143
```

Comment:
Aren't you currently retrieving gtest from msvc16? It should be added to this msvc17 list.

Reply:
Solved after offline discussion